lib/App/DuckPAN/Perl.pm

```diff
@@ -191,16 +191,42 @@ sub duckpan_install {
 	}
 }
 
+my @POTENTIAL_AUTHORS = qw|
+    ALOHAAS
+    ANDREY_P
+    BRAD
+    BRIANS
+    CAINE
+    CRAZEDPSYC
+    DDGC
+    DYLANLL
+    GETTY
```

Comment:
If there's nothing released by these people that we currently use, we can probably remove some of these names. Though it doesn't really hurt to keep them around?

- GETTY
- DYLANLL
- CRAZEDPSYC
- SDOUGBROWN